tests/circleci_pipelines/resources/.circleci/config.yml

```diff
@@ -84,6 +106,37 @@ jobs:
           command: |
               echo ${CIRCLE_PR_REPONAME}
           name: "Echo the PR Reponame"
+
+  test-linux-image:
+    machine:
+      image: ubuntu-1604:202007-01
+    steps:
+      - some-step
+
+  test-macos-image:
+    macos:
+      xcode: 9.4.1
+    steps:
+      - some-step
+
+  test-machine-default:
+    executor:
+      name: win/default-executor
+    steps:
+      - some-step
+
+  test-executor-machine-server-2019:
+    executor:
+      name: win/server-2019
+    steps:
+      - some-step
+
+  test-executor-machine-server-2019-cuda:
+    executor:
+      name: win/windows-server-2019-cuda
+    steps:
+      - some-step
```

Comment:
same here, all are CircleCI internal machine names, nothing we can actually scan